src/services/payments/payments.py

```
@@ -136,14 +149,145 @@
+        price = line_item.get("price", {})
+        product = price.get("product")
+
+        if product:
+            product_obj = stripe.Product.retrieve(
+                product, api_key=self.secret_key
+            )
+            order_item_id = int(
+                product_obj.get("metadata", {}).get("order_item_id")
+            )
+
+        return order_item_id, price["unit_amount"]
```

Pylint (3.3.7)

[error] Possibly using variable 'order_item_id' before assignment

(E0606)